pkg/settings/migration.go

```diff
@@ -126,6 +130,14 @@ func (r *Migration) Load() (err error) {
 	if r.VddkJobActiveDeadline, err = getPositiveEnvLimit(VddkJobActiveDeadline, 300); err != nil {
 		return liberr.Wrap(err)
 	}
+	r.VirtV2vExtraArgs = "[]"
+	if val, found := os.LookupEnv(VirtV2vExtraArgs); found && len(val) > 0 {
```

Comment:
i would prefer to check if the env var exists, if it does, i wouldn't check the length of it
in case we don't have this environmental var, i wouldn't send it to the conversion pod and within the conversion pod if it doesn't exist - do nothing.

in my opinion it will save troubleshooting and the extra marshal/unmarshal. but it also a matter of taste

Reply:
> i would prefer to check if the env var exists, if it does, i wouldn't check the length of it

the reason for checking the length there is also not to fail when the user sets the extra arguments to an empty string